crates/solver/src/settlement_submission/flashbots_settlement.rs

```diff
@@ -258,7 +267,7 @@ impl<'a> FlashbotsSolutionSubmitter<'a> {
                         tracing::warn!("flashbots cancellation request not sent: {:?}", err);
                     }
                 }
-                return anyhow!("flashbots failed simulation: {}", err);
+                return anyhow::Error::new(err).context("flashbots failed simulation");
```

Comment:
ubernit:
```suggestion
                return err.context("flashbots failed simulation");
```

(requres:
```rust
use anyhow::Context as _;
```

Reply:
I tried this but (even with the import) was getting:

> error[E0599]: no method named `context` found for struct `MethodError` in the current scope
   --> crates/solver/src/settlement_submission/flashbots_settlement.rs:271:28
    |
271 |                 return err.context("flashbots failed simulation");
    |                            ^^^^^^^ method not found in `MethodError`

I think I need to convert the MethodError specifically into an anyhow Error before using context.